test/Streamly/Test/Common/Array.hs

```diff
@@ -41,10 +39,11 @@ import qualified Streamly.Internal.Data.Array.Prim.Types as A
 import qualified Streamly.Internal.Data.Stream.IsStream as IP
 type Array = A.Array
 #else
-import qualified Streamly.Internal.Data.Array as A
+import qualified Streamly.Internal.Data.Array.Storable.Foreign as A
```

Comment:
This is incorrect. We already test `Streamly.Internal.Data.Array.Storable.Foreign` when `-DTEST_ARRAY` is used.